templates/terraform/resource.erb

```diff
@@ -309,14 +344,50 @@ func resource<%= resource_name -%>Import(d *schema.ResourceData, meta interface{
 		return nil, fmt.Errorf("Error constructing id: %s", err)
 	}
 	d.SetId(id)
+  <%= lines(compile(object.custom_code.post_import)) if object.custom_code.post_import -%>
 
 	return []*schema.ResourceData{d}, nil
+<% end -%>
 }
 
 <% properties.each do |prop| -%>
+  <% if object.custom_code.custom_flatten.key? prop.name -%>
```

Comment:
This does not work for nested properties.

Consider moving anything this to the properties themselves.

Reply:
Done, agreed.